src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        String color = new ColorSupplier().getRandomColor();
+        ball.setColor(color);
+        ball.setNumber(new Random().nextInt(100));
+        return ball;

```

Comment:
ColorSupplier and Random should be a class-fields
```suggestion
    private final ColorSupplier colorSupplier =  new ColorSupplier();
    private final Random random =  new Random();
    
    public Ball getRandomBall() {
        Ball ball = new Ball();
        String color = colorSupplier.getRandomColor();
        ball.setColor(color);
        ball.setNumber(random.nextInt(100));
        return ball;
```